packages/babel-plugin-minify-simplify/README.md

```diff
@@ -2,9 +2,10 @@
 
 This plugin will transform code in mainly two ways:
 
-1. Reduce as much statements as possible into expressions
+1\. Reduce as much statements as possible into expressions
```

Comment:
Hmm. It's not in the readme on github as well. Cool ... Then we can simply use unordered lists instead of ordered ... Numbering isn't important